### PR TITLE
Alteryx integration

### PR DESCRIPTION
**Pull request by vtols**

### CODE REVIEW

**Comment by vtols** [comment]
@kghamilton89, please, don't make commits to this branch for now as the document is incomplete :disappointed:. I need to proof-read the text by myself before I give it to you.

**Comment by rodionos** [comment]
3. 

```
Alteryx Designer is a graphical design environment to create and edit ETL (Extract, Transform, Load) workflows. The following guide includes examples of loading time series data from the Axibase Time Series Database (ATSD), calculating derived time series in Alteryx and storing the results back in ATSD.
```

**Comment by rodionos** [comment]
4.

```
The series contain the Consumer Price Index (CPI) for each category of items in a consumer's basket as well as a weight for each category in the CPI basket. The weights are stored as fractions of 1000. The CPI is tracked from 2013 to 2017 and uses Year 2016 values as the baseline. Weight values are available only for 2017. The underlying data is available in the following Excel file.

To calculate a weighted inflation index we need to multiply the CPI of each category by its weight divided by 1000 and sum the products.
```

File: integration/alteryx/README.md [hunk 1]
```diff
@@ -3,70 +3,70 @@
...
-  follow instructions in [ODBC-JDBC Bridge](../odbc/README.md), and in order to
+- In case you have not configured the ODBC connection with ATSD before, you need to
+  follow the instructions shown here: [ODBC-JDBC Bridge](../odbc/README.md), and in order to
   complete the [Configure ODBC Data Source](../odbc/README.md#configure-odbc-data-source)
-  step of these instructions you can open the **ODBC Data Source Administrator**
-  window by pressing **ODBC Admin** button in the **ODBC Connection** dialog. You will
-  want to use the `tables` connection property in DSN URL set to some
-  value. It allows you choose a table from the metrics list that satisfy some pattern.
-  For example if we use this URL
+  step of these instructions you must open the **ODBC Data Source Administrator**
```
Comment (vtols) [inline integration/alteryx/README.md]: kghamilton89, why you did replace _can_ with _must_ here
> ... to complete the ... step of these instructions you _must_ open ... by pressing .,.

User is not obliged to press the button. He has choice, do what is described in _ODBJ-JDBC_ document, or press the button as shortcut.

File: integration/alteryx/README.md [hunk 2]
```diff
@@ -111,27 +111,27 @@ The final arrangement of the tools in the workflow will be:
...
    ![](images/filter.png)
 
    ![](images/true_output.png)
 
-4. **Filter** tool. Same as previous.
+4. **Filter** tool. Follow the same procedure as above.
 
-5. **Join** tool. Here it joins by `entity` and `tags.category` fields.
-   Some fields are exluded from thee join results. `value` fields for
+5. **Join** tool. This tool joins the `entity` and `tags.category` fields.
```
Comment (vtols) [inline integration/alteryx/README.md]: It doesn't joins entity and tags, it joins **BY** them. Here's example from Wiki
image

File: integration/alteryx/README.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+- Table names visible in the **Tables** tab satisfy the `tables` pattern specified in the DSN URL. Click the **Refresh** button to reload the list, if neccessary.
+
+  ![](images/metrics_list.png)
+
+- In the **Visual Query Tab**, you can specify particular columns and add optional
+  sorting and grouping to preview and prepare your
+  data before processing it in workflow. Below is a SQL query and a corresponding configuration.
+
+  ```sql
+   Select
```
Comment (rodionos) [inline integration/alteryx/README.md]: Rewrite query to adhere to our format used in ATSD SQL docs: capitalized reserved keywords, two-space step-wise indentation.

File: integration/alteryx/README.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+To load the data into ATSD, login into the database web interface and submit these commands on the 
+**Metrics→Data Entry** page.
+
+![](images/metrics_entry.png)
+
+## Prerequisites
+
+- Install Alteryx Designer
+- Install [ODBJ-JDBC gateway](../odbc/README.md).
```
Comment (rodionos) [inline integration/alteryx/README.md]: fix type

File: integration/alteryx/README.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+- Add `tables` property into the DSN URL to filter metrics by name in the Query Builder. For example, `tables=*` exposes all metrics whereas `tables=infla*` shows all metrics that start with 'infla' substring.
+
+  ```text
+  jdbc:axibase:atsd:ATSD_HOST:8443;tables=*
+  ```
+  
+> Read more about the `table` property in the [JDBC driver](https://github.com/axibase/atsd-jdbc#jdbc-connection-properties-supported-by-driver) documentation.
+
+- Unckeck the **Strip Quote** flag and press **OK**.
```
Comment (rodionos) [inline integration/alteryx/README.md]: fix type and run spell checker on this version. Replace screenshot showing url for the latest driver version.

File: integration/alteryx/README.md [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+  ```
+
+  ![](images/visual_builder.png)
+
+- The **SQL Editor** allows you to review and modify pre-built queries or write
+  your own queries. 
+  
+- The **Test Query** button strips the `WHERE` clause from the query and sends only the remaining `SELECT` expression to the database for validation.
+
+  ![](images/sql_editor.png)
```
Comment (rodionos) [inline integration/alteryx/README.md]: Replace screenshot showing query with capitalized reserved keywords and step-wise indentation.

File: integration/alteryx/README.md [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+   ![](images/sort.png)
+
+9. **Formula** tool. Add an `entity` column with the **Formula** tool. Name it
+   `bls.gov`. The default data type `V_WString` is not supported yet by the ATSD JDBC driver,
+   use `String` or `WString` (for Unicode) instead.
+
+   ![](images/entity.png)
+
+10. **Output Data** tool. Choose ODBC Connection as before and enter a name for
+    the _existing_ metric, in this case `inflation.cpi.composite.price`. Create a new metric in ATSD if necessary.
```
Comment (rodionos) [inline integration/alteryx/README.md]: Is this still the case? Check with the latest driver if pre-creating the metric is still necessary.

File: integration/alteryx/README.md (hunk 3 above)
Comment (vtols) [inline integration/alteryx/README.md]: It is the query that Alteryx builds, not handwritten

File: integration/alteryx/README.md (hunk 6 above)
Comment (vtols) [inline integration/alteryx/README.md]: Screenshot shows the query that was built by Alteryx

File: integration/alteryx/README.md (hunk 5 above)
Comment (vtols) [inline integration/alteryx/README.md]: Which version we consider as latest?

**Pull request merged**